### PR TITLE
Fixed broken link in Faq section

### PR DESCRIPTION
**Pull request by shelar1423**
Fixes Issue  : #250

### CODE REVIEW

**Comment by shelar1423** [comment]
@sumitsaurabh927 fixed it, let me know if changes are required .Thank you

File: faqs/questions.mdx
Comment (sumitsaurabh927) [inline faqs/questions.mdx:1]: can you please change it like this: '...or digested with other notifications into'?

**Review by sumitsaurabh927 — CHANGES_REQUESTED** [review]

**Comment by shelar1423** [comment]
An event is every call to our [trigger API or digested with other notifications into ]
 @sumitsaurabh927 want me to do this ?

**Comment by shelar1423** [comment]
Made the changes 🙏 @sumitsaurabh927

**Review by unicodeveloper — APPROVED** [review]

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**